### ISSUE TITLE
BUG: Auto suggest does not work on the locations dropdown

### ISSUE THREAD

**Issue by mandily-p** [BCcampus/eypd#579]
### Description 

[Description of the bug or feature]

**Expected behaviour** 

[What you expect/want to happen]

**Actual behaviour** 

[What actually happened]

### Steps to reproduce the problem

[First Step, Second Step, etc]

### System Information

- EYPD: [Enter Version here] 
- WordPress: [Enter WP Version number here]

**Comment by kathreenriel** [comment]
Yes, will include in the Sprint. It's a mystery to me why this stopped working.

**Comment by bdolor** [comment]
resolved via https://github.com/BCcampus/eypd/commit/a2d4d62638d8ae02f751a674bd048328c65b85de

**Comment by mandily-p** [comment]
I can't access the post event page on cert to validate this. @paulagaube @kkilbey Are either of you able to see the post event page?

![screen shot 2018-11-27 at 3 11 10 pm](https://user-images.githubusercontent.com/20425828/49117790-d8033c00-f256-11e8-87a7-544195c05967.png)

**Comment by bdolor** [comment]
this can also be validated in the dashboard...the page not showing is unrelated to the autocomplete feature.

**Comment by kathreenriel** [comment]
@mandily-p @kkilbey @paulagaube I just posted an event and the auto-suggest locations works.

**Comment by kkilbey** [comment]
Auto-suggest menu works for locations on the Post an Event page on prod. However, it does not work for the location field in the Sign Up page on either cert or prod (part of original ticket description).

**Comment by bdolor** [comment]
The location field on the sign up page has never had auto-complete...

**Comment by kkilbey** [comment]
@bdolor Ah, so disregard Steps to Reproduce the Problem - Version 1 (above)? If so, this is done on prod!
